test/typescript-generator/script.test.js

```diff
@@ -93,9 +93,9 @@ describe("a Script", () => {
     script.importDefault(coder);
 
     expect(script.importStatements()).toStrictEqual([
-      'import { Account0 } from "./export-from-me.js";',
-      'import type { Account1 } from "./export-from-me.js";',
-      'import Account2 from "./export-from-me.js";',
+      'import { Account0 } from "export-from-me.js";',
+      'import type { Account1 } from "export-from-me.js";',
+      'import Account2 from "export-from-me.js";',
```

Comment:
We still want the `./`s here. This test should not change. We need another test where we're importing from a parent directory and make sure it's `../` instead of `./../`.

Reply:
done